test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,22 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  it("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe('Fizz')
```

Comment:
the test uses "" double quotation marks, I would suggest to use "" throughout for better consistency